test/fixtures/get-orderbook-response.json

```diff
@@ -324,9 +324,9 @@
           "value": "0.5",
           "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
         },
-        "totalPrice": {
+        "price": {
```

Comment:
Oh yeah, I was thinking in terms of javascript math. I guess we could do it losslessly with the bignumber library. Users would have to use our math libraries to reproduce the calculations anyways, so I guess it's reasonable to assume they won't be using javascript math if they want accurate results.